test/imports-requiring-flow_test.js

```diff
@@ -96,6 +96,16 @@ ruleTester.run("imports-requiring-flow", rule, {
                 },
             ],
         },
+        {
+            code: importBarModFlow,
+            filename: path.join(rootDir, "src/package-1/foobar.js"),
+            options: [
+                {
+                    modules: ["src/package-2/"],
```

Comment:
It's a little weird to be passing directories to something called `modules`.  I don't have a better name for it though.

Reply:
Directories are modules, though. Node doesn't really care. If it doesn't find a package.json and it's a directory, it would just load index.js from the dir. I don't think there's any need to rename anything here.